types/react-router-breadcrumbs-hoc/index.d.ts

```diff
@@ -13,9 +13,15 @@ export interface Options {
   pathSection?: string;
 }
 
+export interface MatchOptions {
+  exact?: boolean;
+  strict?: boolean;
```

Comment:
Thanks for fixing this! 🙌 

From what I can tell [`sensitive?: boolean;`](https://github.com/ReactTraining/react-router/blob/d28d46dce08a5756a085f7e5eebb5169ea59e40b/packages/react-router/modules/matchPath.js#L31) should also be added here for completeness.

Reply:
nps @wkillerud!

Weird that they haven't included that param in their official [docs](https://reacttraining.com/react-router/web/api/matchPath) yet, but I will include it here for completeness.

Reply:
Done!